analysis/wwz/yld_dicts_for_comp.py

```diff
@@ -131,6 +131,4 @@
1648351], 'sr_4l_bdt_of_coarse_2': [2.1520825818931417, 0.012938799652013418], 'sr_4l_bdt_of_coarse_3': [0.23703764701964325, 0.0012782487431890325], 'sr_4l_bdt_of_coarse_4': [5.5253637279979735, 1.9955159983570208], 'cr_4l_btag_of': [13.040770225915475, 0.16000752223005102], 'cr_4l_btag_sf_offZ_met80': [6.45631125677942, 0.03829502906137419], 'cr_4l_sf': [422.24968122489867, 0.3679695946306091], 'sr_sf_all_cutbased': [3.852711360191054, 0.06962659428668722], 'sr_of_all_cutbased': [2.1618580851840052, 0.0109838198618282], 'sr_all_cutbased': [6.014569445375059, 0.08061041414851541], 'sr_sf_all_bdt': [28.503670390543256, 0.09915438409822516], 'sr_of_all_bdt': [8.149567908964862, 2.010564421716388], 'sr_all_bdt': [36.65323829950811, 2.1097188058146132], 'sr_sf_all_bdtCoarse': [28.50367039054326, 0.09915438409822519], 'sr_of_all_bdtCoarse': [8.149567908964862, 2.010564421716388], 'sr_all_bdtCoarse': [36.653238299508125, 2.1097188058146132]}, 'Zmetric': {'sr_4l_sf_A': [0.6858392374299204, None], 'sr_4l_sf_B': [0.48354154826384343, None], 'sr_4l_sf_C': [0.2523047049053227, None], 'sr_4l_of_1': [1.0415606239812316, None], 'sr_4l_of_2': [0.6588899475004611, None], 'sr_4l_of_3': [0.5100634407405644, None], 'sr_4l_of_4': [0.9134606969476136, None], 'sr_4l_bdt_sf_1': [0.6014283989410638, None], 'sr_4l_bdt_sf_2': [0.39594158951167757, None], 'sr_4l_bdt_sf_3': [0.3484605706423991, None], 'sr_4l_bdt_sf_4': [0.06396521106238846, None], 'sr_4l_bdt_sf_5': [0.6920688294347158, None], 'sr_4l_bdt_sf_6': [0.2738360502876695, None], 'sr_4l_bdt_sf_7': [0.14043520341542284, None], 'sr_4l_bdt_sf_8': [0.05693512982642242, None], 'sr_4l_bdt_of_1': [0.48778286047825886, None], 'sr_4l_bdt_of_2': [0.8832453134788101, None], 'sr_4l_bdt_of_3': [0.7061203998508114, None], 'sr_4l_bdt_of_4': [0.3695315707429991, None], 'sr_4l_bdt_of_5': [1.3820360252848263, None], 'sr_4l_bdt_of_6': [0.6370829517071711, None], 'sr_4l_bdt_of_7': [0.7373091063982443, None], 'sr_4l_bdt_of_8': [0.14224750213284715, None], 'sr_4l_bdt_sf_coarse_1': [0.7009237124893868, None], 'sr_4l_bdt_sf_coarse_2': [0.16267797246784924, None], 'sr_4l_bdt_sf_coarse_3': [0.6321848204373318, None], 'sr_4l_bdt_sf_coarse_4': [0.10184534983170086, None], 'sr_4l_bdt_of_coarse_1': [0.9528041471262126, None], 'sr_4l_bdt_of_coarse_2': [0.7264357894264413, None], 'sr_4l_bdt_of_coarse_3': [1.4884176665941733, None], 'sr_4l_bdt_of_coarse_4': [0.31696312135532484, None], 'cr_4l_btag_of': [0.16857469943895115, None], 'cr_4l_btag_sf_offZ_met80': [0.10880440573578155, None], 'cr_4l_sf': [0.011165287596054951, None], 'sr_sf_all_cutbased': [0.8762679684966302, None], 'sr_of_all_cutbased': [1.6166507522632885, None], 'sr_all_cutbased': [1.8388597574058407, None], 'sr_sf_all_bdt': [1.1049485424016356, None], 'sr_of_all_bdt': [2.1290775787654397, None], 'sr_all_bdt': [2.398725206804065, None], 'sr_sf_all_bdtCoarse': [0.9632176783727954, None], 'sr_of_all_bdtCoarse': [1.936852464559312, None], 'sr_all_bdtCoarse': [2.1631425670536277, None]}}
```

Comment:
Is this just 22, or the scaled 22+23 numbers?

Reply:
I think I may have had this as a scaled 2022, but just 2022 would be better (updated)